kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/TestUtils.java

```diff
@@ -26,21 +33,24 @@ public static FileAttribute<Set<PosixFilePermission>> getDefaultPosixFilePermiss
     }
 
     /**
-     * Concat string if value don't exist.
+     * Gets resources path.
      *
-     * @param originalString the original String
-     * @param stringToConcat the string to concat
-     * @param stringSeparator the string separator
-     * @return the string
+     * @param fileName the file name
+     * @return the resources path
      */
-    public static String concatStringIfValueDontExist(String originalString, String stringToConcat, String stringSeparator) {
-        if (originalString.isEmpty() || originalString.isBlank()) {
-            stringSeparator = "";
+    @NonNull
+    public static Path getResourcesPath(String fileName) {
```

Comment:
Classpath resources might be within Jar files, so Path is an inappropriate choice of return type.  URL or URI would be better.